src/CommonDBChild.php

```diff
@@ -865,11 +865,12 @@ public static function showAddChildButtonForItemForm(
             $lower_name         = strtolower(get_called_class());
             $child_count_js_var = htmlescape('nb' . $lower_name . 's');
             $div_id             = htmlescape("add_" . $lower_name . "_to_" . $item->getType() . "_" . $items_id);
+            $add_label          = sprintf(__('Add a new %s'), call_user_func([get_called_class(), 'getTypeName']));
```

Comment:
This seems weird, can't you just do this ?

```suggestion
            $add_label          = sprintf(__('Add a new %s'), static::getTypeName());
```

Reply:
Also, we probably need `__s` instead of `__` to be safe.